.buildkite/pipeline.test.yml

```diff
@@ -1,3 +1,5 @@
+#ci:group=Minimal dependency tests
```

Comment:
I do not know what these tests are, but the tests listed in this file does not say "min deps tests"?

Reply:
Yeah this was confusing to me as well, but this seems to be the intended use case... 

> pipeline.test.yml contains jobs that only require an installed Ray and a small subset of dependencies, notably exlcuding ML libraries such as Tensorflow or Torch. The tests are run on the test.Dockerfile image.

I don't know the logic for why the minimal dependency tests are in the other pipelines, given this description.